### ISSUE TITLE
I have a problem :(

### ISSUE THREAD

**Issue by najibghadri** [feross/simple-peer#761]
I built this game half a year ago
[https://quarantime.io/](https://quarantime.io/)
Everything worked fine until around a month ago. 
I updated simplepeer to the latest version but didn't fix the issue.
Just open the same room twice.
On the same machine you can get connected with video and everything.
But even on the same network, different machines ERR_CONNECTION_FAILURE.
I tried many things:
Allow/disable trickle and halftrickle
Changed up stun servers many times.

You can see logs in the console for simple peer events with signals logged too.
I have been working on this for two weeks now. Please help :( Thank you!

**Comment by najibghadri** [comment]
I am using these servers now
```
          { urls: 'stun:stun.l.google.com:19302' }, 
          { urls: 'stun:global.stun.twilio.com:3478?transport=udp' },
          { urls: "stun:stun.stunprotocol.org:3478" },
          { urls: "stun:stun.sipnet.ru:3478" },
          { urls: "stun:stun1.l.google.com:19302" },
```

**Comment by feross** [comment]
You need TURN servers or else some of your connections will fail. That's just how WebRTC works.

**Comment by najibghadri** [comment]
Thank you for answering @feross ! I will try that with coturn, but somehow I didn't need that. Actually I deleted the issue yesterday because I tried something and it worked: between different machines it worked. So from my mac to my phone on chrome it doesn't work but between Linux chrome and phone it works. So I realized it's not simple peer issue, but most probability api change, hence adapter.
I got add ice candidate error..

But I don't think anyone can help honestly. Idk.
